scripts/firedrake-install

```diff
@@ -482,7 +485,7 @@ if travis:
 
 def install(package):
     log.info("Installing %s" % package)
-    if travis and travis_cache_dir and package in ["petsc/", "petsc4py/"]:
+    if travis and travis_cache_dir and package in ["petsc/", "petsc4py/", "slepc/", "slepc4py/"]:
```

Comment:
I don't think you mean this. This is the magic which causes petsc to not get rebuilt on travis. This is only needed because building petsc takes so long.